src/app/config/feature-manager/captcha.config.ts

```diff
@@ -1,21 +1,25 @@
-import { FeatureNames, RegisterableFeature } from './types'
+import convict, { Schema } from 'convict'
```

Comment:
doesn't need to be in this PR, but should we move/inline this file into config itself?

Reply:
yes that's the plan! I think it's neater to keep them in their own folder, so I might just rename the `feature-manager` folder to `features`